src/norm_flows.jl

```diff
@@ -0,0 +1,130 @@
+using Distributions
+using LinearAlgebra
+using Random
+using StatsFuns: softplus
+using Roots # for inverse
+
+################################################################################
+#                            Planar and Radial Flows                           #
+#             Ref: Variational Inference with Normalizing Flows,               #
+#               D. Rezende, S. Mohamed(2015) arXiv:1505.05770                  #
+################################################################################
+
+(b::Bijector)(x) = transform(b, x)
```

Comment:
Could you make this
```julia
(b::PlanarFlow)(x) = transform(b, x)
(b::RadialFlow)(x) = transform(b, x)
```

Otherwise, a user might try to implement his/her own `Bijector`, forget to implement `(b::UserBijector)` and get an error saying no function named `transform(b:UserBijector, x)` exists; we'd like the user to get an error saying no function `(b::UserBijector)(x)` exists.

Reply:
If `UserBijector<:Bijector`, Wouldn't the need to define `(b::UserBijector)` be removed as long as the user defines `transform(b, x)`

Reply:
Yup, but the decision is to _not_ implement `transform(b, x)` but instead implement the call:)
If `UserBijector <: Bijector` then you have to implement
```julia
(b::UserBijector)(x) = ... # the actual transformation
(ib::Inversed{<: UserBijector})(y) = .. # inverse transformation
logabsdetjac(b::UserBijector, x) = ...
```
So not necessary to implement `transform` at all.

See https://github.com/TuringLang/Bijectors.jl/pull/27#discussion_r314322164

Reply:
Okay, I will make the changes. Thanks!